packages/react-select-async-paginate/package.json

```diff
@@ -40,8 +40,8 @@
   "dependencies": {
     "@babel/runtime": "^7.16.5",
     "@seznam/compose-react-refs": "^1.0.6",
-    "react-is-mounted-hook": "^1.1.2",
-    "sleep-promise": "^9.1.0"
+    "sleep-promise": "^9.1.0",
+    "use-is-mounted-ref": "^1.5.0"
```

Comment:
Why is this package not being used in code? Is it not suitable alternative of `react-is-mounted-hook`?

Reply:
It was causing `yarn validate` to fail so I opted to use an internal version instead.